docker/demo/config/test-suite/complex-dag-mor.yaml

```diff
@@ -13,107 +13,72 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-dag_name: complex-dag-mor.yaml
+dag_name: cow-long-running-example.yaml
 dag_rounds: 1
-dag_intermittent_delay_mins: 10
+dag_intermittent_delay_mins: 1
 dag_content:
   first_insert:
     config:
-      record_size: 70000
+      record_size: 100
```

Comment:
Record size of 100 bytes ?

Reply:
anyways, w/o complex schema, we are not generating records of size 7000 bytes. So, thought will keep it to some sane value.